s3parq/publish_redshift.py

```diff
@@ -178,7 +178,7 @@ def _datatype_mapper(columns: dict) -> dict:
     return f"({sql_statement[:-2]})"  # Slice off the last space and comma
 
 
-def create_schema(schema_name: str, db_name: str, iam_role: str, session_helper: SessionHelper) -> None:
+def create_schema(schema_name: str, db_name: str, iam_role: str, session_helper: SessionHelper, read_access_user=None) -> None:
```

Comment:
Given that this is called directly from publish, can we add an optional `read_access_user` to our `redshift_params` in `publish` and `custom_publish`? That way this can be used in existing workflows

Reply:
yes!  good catch